geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -505,7 +509,7 @@ public void testScenario5() throws Exception {
         String current_fullpath = jtaObj.currRegion.getFullPath();
         assertEquals(
             "failed retrieving current fullpath after rollback, fullpath is: " + current_fullpath,
-            "/" + DEFAULT_RGN + "/region1", current_fullpath);
+            SEPARATOR + DEFAULT_RGN + "/region1", current_fullpath);
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?